docs/src/conf.py

```diff
@@ -85,21 +85,10 @@ def autolog(message):
 author = "Iris Developers"
 
 # The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-
-# The short X.Y version.
-if iris.__version__ == "dev":
-    version = "dev"
-else:
-    # major.minor.patch-dev -> major.minor.patch
-    version = ".".join(iris.__version__.split("-")[0].split(".")[:3])
-
-# The full version, including alpha/beta/rc tags.
-release = iris.__version__
+# |version|, also used in various other places throughout the built documents.
 
+version = iris.__version__
```

Comment:
This doesn't align with [`setuptools_scm` recommendations](https://github.com/pypa/setuptools_scm#usage-from-sphinx).

Reply:
Done :+1:

Reply:
How come we no longer have a `release` variable?

Reply:
I figure it's just not used anymore. I can't see anywhere in the docs that references it.

Perhaps @tkknight might know this one :thinking:

Reply:
The docs build without any warnings and they render fine, so that's good enough for me

Reply:
Version displayed in latest docs is now "v0.1.dev72+dirty".  Was that expected?

Reply:
I got `3.3.0` when I tested it. @bjlittle any ideas?